app/controllers/assessment/assessments_controller.rb

```diff
@@ -102,6 +106,77 @@ def index
     end
   end
 
+  # Export all completed assessments of the given type as a zip file of PDFs.
+  def dump_pdfs
+    assessment_type = params[:type]
+
+    # Abort if PDF export is not enabled.
+    @pdf_export = @course.pdf_export(assessment_type).display
```

Comment:
@raymondtangsc Will push out a commit shortly to remove the unnecessary use of an instance variable in `dump_pdfs`.